src/validationinterface.cpp

```diff
@@ -177,18 +177,17 @@ void SyncWithValidationInterfaceQueue()
 // evaluating arguments when logging is not enabled.
 //
 // NOTE: The lambda captures all local variables by value.
-#define ENQUEUE_AND_LOG_EVENT(event, fmt, name, ...)           \
-    do {                                                       \
-        auto local_name = (name);                              \
-        LOG_EVENT("Enqueuing " fmt, local_name, __VA_ARGS__);  \
-        m_internals->m_schedulerClient.AddToProcessQueue([=] { \
-            LOG_EVENT(fmt, local_name, __VA_ARGS__);           \
-            event();                                           \
-        });                                                    \
+#define ENQUEUE_AND_LOG_EVENT(category, level, event, fmt, ...)    \
```

Comment:
No real need to include the `level` as a param, I think. Hardcoding it would make things a bit less noisy.

Reply:
This PR always sets `level` to `Debug`, so we don't need it here. But then any later tweak in log levels would have to edit this macro again.